jstp/src/main/java/com/metarhia/jstp/JSTP.java

```diff
@@ -7,17 +7,32 @@
 
 /**
  * Common JSTP interface
+ *
  * Created by lidaamber on 18.06.16.
```

Comment:
I think we don't need the person and date at all here, the description alone would be enough. One may use git to identify the one who created or modified it.